TenancyInformationApi/V1/Infrastructure/UhContext.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace TenancyInformationApi.V1.Infrastructure
+{
+    public class UhContext : DbContext
+    {
+        public UhContext(DbContextOptions options) : base(options)
+        {
+            var tenancyAgreements = UhTenancyAgreements
+                .Include(t => t.UhAgreementType)
+                .Include(t => t.UhTenureType)
+                .ToList();
+        }
+
+        public DbSet<UhTenancyAgreement> UhTenancyAgreements { get; set; }
+    }
+
+    [Table("lookup")]
```

Comment:
If the table is in a schema which isn't `public`, this also needs to be specified here I think something like: `[Table("name"), Schema("dbo")]`